src/main/java/com/nekomaster1000/infernalexp/InfernalExpansion.java

```diff
@@ -208,6 +212,32 @@ protected ItemStack dispenseStack(IBlockSource source, ItemStack stack) {
 
     private void clientSetup(final FMLClientSetupEvent event) {
         DistExecutor.unsafeRunWhenOn(Dist.CLIENT, () -> InfernalExpansionClient::init);
+
+        loadInfernalResources();
+    }
+
+    private static void loadInfernalResources() {
+        // Creates file location for resource pack
+        File dir = new File(".", "resourcepacks");
```

Comment:
I believe there is a method for retrieving the working directory for Minecraft, it should be used instead of the `.` directory.